admin_guide/preventing_node_placement.adoc

```diff
@@ -0,0 +1,101 @@
+= Preventing Specific Node Placement
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+As a cluster administrator, you can prevent users from targeting specific nodes
+when scheduling pods using labels and node selectors, as well as the node
+blacklist setting.  
+
+== Constraining Pod Placement
+
+Adding a label to the `nodeSelectorLabelBlacklist` section of a master
+configuration file prevents pods from being created if their labels match any
+values in the `nodeSelectorLabelBlacklist`. This means, if no node selector is
+available, then the scheduler can still place a pod on a node with a blacklisted
```

Comment:
@bfallonf This sentence seems unnecessary, and the wording is ambiguous and confusing. The example below explains this particular behavior of the plugin clearly enough, with the benefit of more complete context.